features/support/hooks.js

```diff
@@ -37,33 +36,31 @@ module.exports = function () {
     // application server mock
     this.app = makeExpressApp(this)
     this._server = http.createServer(this.app)
-    var primusOpts = {
+    const primusOpts = {
```

Comment:
I think of defining constants as things at the top level of the document that you use everywhere else. var/let should live inside functions where the value might change from execution to execution.